content/explanations/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Explanations"
```

Comment:
@osterman I'm thinking we might want to call this top level section "Introduction" considering "Explanations" feels awkward to me. Thoughts?

Reply:
Yes! I think "explanations" sound awkward too.